META-INF/plugin.xml

```diff
@@ -635,6 +635,10 @@ We keep delivering funded results =)
         shortName="UsageOfSilenceOperatorInspection"              displayName="Usage of the silence operator"
         groupName="Code Style"                                    enabledByDefault="true" level="WARNING"
         implementationClass="com.kalessil.phpStorm.phpInspectionsEA.inspectors.codeSmell.UsageOfSilenceOperatorInspector"/>
+    <localInspection language="PHP" groupPath="PHP,Php Inspections (EA Extended)"
+        shortName="MagicNumberInspection"                         displayName="Magic number usage"
+        groupName="Code Style"                                    enabledByDefault="true" level="WEAK WARNING"
```

Comment:
What about magic numbers as 86400, 3600, 60, 24, 1970 - should we consider them as well? I'm asking because you probably gave the inspection test run for some weeks.

And I need you advise if I could pick up another PR instead, which is smaller?